Documentation/gettingstarted/bgp-control-plane.rst

```diff
@@ -0,0 +1,326 @@
+instantiated virtual router on a particular Cilium node is explained
+in*\ `Virtual Router Attributes <#Virtual%20Router%20Attributes>`__
+
+Creating a BGP Topology
+-----------------------
+
+Rules
+~~~~~
+
+The following rules must be adhered to for a ``CiliumBGPPeeringPolicy``
+to correctly apply to a node.
+
+-  Only a single ``CiliumBGPPeeringPolicy`` can apply to a ``Cilium``
+   node.
+
+   -  If the ``BGP Control Plane`` on a node iterates through the
+      ``CiliumBGPPeeringPolicy`` CRs currently written to the cluster
+      and discovers (n > 1) policies match its labels, it will return an
+      error and remove any existing BGP sessions. Only (n == 1) policies
+      **must** match a node’s label sets.
+   -  Administrators should test a new BGP topology in a staging
+      environment before making permanent changes in production.
+
+The above diagram demonstrates the aforementioned error case. Cilium-01
+will not apply any ``CiliumBGPPeeringPolicy`` since both policies match
+its label set.
+
+-  Within a ``CiliumBGPPeeringPolicy`` each ``CiliumBGPVirtualRouter``
+   defined must have a unique ``localASN`` field.
+
+   -  A node cannot host two or more logical routers with the same local
+      ASN. Local ASNs are used as unique keys for a
+      logical router.
+   -  A node can define the remote ASN on a per-neighbor basis to
+      mitigate this scenario. See ``CiliumBGPNeighbor`` CR
+      sub-structure.
+
+-  IPv6 single stack deployments **must** set an IPv4 encoded
+   ``routerID`` field in each defined ``CiliumBGPVirtualRouter`` object
+   within a ``CiliumBGPPeeringPolicy``
+
+   -  Cilium running on a IPv6 single stack cluster cannot reliably
+      generate a unique 32 bit BGP router ID, as it defines no unique
+      IPv4 addresses for the node. The administrator must define these
+      IDs manually or an error applying the policy will occur.
+   -  This is explained further in `Virtual Router
+      Attributes <#Virtual%20Router%20Attributes>`__
+
+Defining topology
+~~~~~~~~~~~~~~~~~
+
+Within a ``CiliumBGPPeeringPolicy`` multiple
+``CiliumBGPVirtualRouter``\ (s) can be defined.
+
+Each one can be thought of as a logical BGP router instance.
+
+Defining more then one ``CiliumBGPVirtualRouter`` in a
+``CiliumBGPVirtualRouter`` creates more then one logical BGP router on
+the hosts which the policy matches.
+
+It is possible to create a single ``CiliumBGPPeeringPolicy`` for all
+nodes by giving each node in a cluster the same label and defining a
+single ``CiliumBGPPeeringPolicy`` which applies to this label.
+
+It is also possible to provide each ``Kubernetes`` node its own
+``CiliumBGPPeeringPolicy`` by giving each node a unique label and
+creating a ``CiliumBGPPeeringPolicy`` for each unique label.
+
+This allows for selecting subsets of nodes which peer to a particular
+BGP router while another subset of nodes peer to a separate BGP router,
+akin to an “AS-per-rack” topology.
```

Comment:
Curly quotes. Have you tried disabling Pandoc's `smart` extension (I suppose it's on by default)? `pandoc -f markdown-smart ...`

Reply:
I tried the "-t plain" flag with pandoc conversion, but it removes *all* mark up including backticks and such..